### DISCUSSION TITLE
did the pull request for fix dependent task bug have been pick up to 3.0

### DISCUSSION THREAD

**Discussion by lordk911** [apache/dolphinscheduler#11439] [Q&A]
before version 2.0.6\2.0.7 there are many bugs about dependent task, have the release version 3.0.0 picked up these pr ?  or will be in next version 3.0.1 ?

**Comment by lordk911** [comment]
@JinyLeeChina do you know about this question?

**Reply by brave-lee** [reply]
PR not merged，pls refer #11204

**Reply by lordk911** [reply]
thanks

**Reply by lordk911** [reply]
@JinyLeeChina wonder to know why the PR still not merged?
